docs/source/migrating/migrate_v2.rst

```diff
@@ -0,0 +1,69 @@
+How to migrate to 2.0.0 from versions pre 1.0.1 
+===============================================
+
+If you previously used older versions of cleanlab, 
+this migration guide can help you update your existing code to work with v2.0.0 in no time!
+Below we outline the major updates and code substitutions to be aware of. 
+The full change-log is listed in the `v2.0.0. Release Notes <https://github.com/cleanlab/cleanlab/releases/tag/v2.0.0>`_.
+
+
+Method and class name changes
+-----------------------------
+
+| **Old:** ``pruning.get_noise_indices(s, psx, prune_method, sorted_index_method, ...)``
+| --> 
+| **New:** ``filter.find_label_issues(labels, pred_probs, filter_by, return_indices_ranked_by, ...)``
```

Comment:
Can we cross-ref all the new class/function/method names, so they are clickable?